### PR TITLE
Grid branch

### PR DESCRIPTION
**Pull request by ruchiragoswami**
grid-branch

Please review this grid branch
There are two separate grids, one with two items, and another with three items

### CODE REVIEW

**Comment by Kevin-Solomon** [comment]
Hey, Ruchi one small suggestion try using your responsive image class in two item grid.

**Review by Kevin-Solomon — APPROVED** [review]

File: grids/grids.css [hunk 1]
```diff
@@ -0,0 +1,33 @@
+.container-grid {
+    width: 50%;
+}
+
+/* for multiple items in grid, uncomment the two lines below */
+.two-item-grid {
+    display: grid;
+    grid-gap: 10px;
+    /* grid-template-columns: repeat(2, 1fr); */
```
Comment (NamrataRaikwar2002) [inline grids/grids.css]: Hey, you can remove these comments.

File: grids/grids.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+/* Three Grid items */
+
+/* for multiple items in grid, uncomment the line below */
+.three-item-grid {
+    display: grid;
+    grid-gap: 10px;
+    grid-template-columns: repeat(3, 1fr);
+    /* grid-template-rows: 0.1fr 1fr; */
+    grid-template-areas:
+        " kelly nick josh"
```
Comment (SaiArunesh) [inline grids/grids.css]: Sorry if I did not get the reference. Neutral naming like item1 item2 item3 is preferrable. And these are supposed to be overridden by the end user not in the library. So have the custom naming overrides in a separate css file that imports grid.css. Easier to publish this way

**Review by SaiArunesh — CHANGES_REQUESTED** [review]
Please refer the above suggestion

**Review by NamrataRaikwar2002 — COMMENTED** [review]
All are fine, nice work Ruchi.

File: grids/grids.css (hunk 2 above)
Comment (ruchiragoswami) [reply grids/grids.css]: item1, item2 are too generic, especially for grid items. These names exist everywhere in our code. Therefore, I prefer to give specific IDs for grid items, so while laying them out in grid template areas, it becomes easier, I feel.

File: grids/grids.css (hunk 1 above)
Comment (ruchiragoswami) [reply grids/grids.css]: Hey Namrata, I need those comments for future. Right now there are two items only in the grid, when we have more items, users can uncomment this and choose the number of columns. I want to leave the comment there. 
Example, if user wants to arrange 15 items in 3 columns, this comment will be useful.

File: grids/grids.css (hunk 2 above)
Comment (ruchiragoswami) [reply grids/grids.css]: Changed anyways!

File: grids/grids.css (hunk 1 above)
Comment (ruchiragoswami) [reply grids/grids.css]: Removed anyways!

**Review by vermasavita — APPROVED** [review]
Good Job!!

**Review by NamrataRaikwar2002 — APPROVED** [review]
good work

**Pull request merged**